README.md

```diff
@@ -1,6 +1,14 @@
-# Awesome Neutralino
+# Awesome Neutralinojs [![Awesome](https://awesome.re/badge.svg)](https://awesome.re)
```

Comment:
Should I rename the repo to match? Didn't realize that the `js` was part of the official name 😅

Reply:
Hey.. mm I think "neutralinojs" looks nice. but you can use repo name as you wish :D .. I suggested "Neutralinojs" since we use that name everywhere in documentation, etc

Reply:
gotcha. I'll probably keep the repo name as is in that case. seems analogous to the fact that Electron can also be referred to as Electron.js 😄